### DISCUSSION TITLE
希望连接时间或次数修改，挂载网盘无法正常访问大音频和视频

### DISCUSSION THREAD

**Discussion by 952362** [AlistGo/alist#6347] [Q&A]
我在 115 网盘里存了一些大文件的音频的视频，其中音频文件存在 1Gb左右一个文件，格式是dsf，在后台添加dsf预览后尝试预览会提示an audio error has soccurred, player will skip forward in 2 seconds
我想知道这个是具体什么原因
然后有一个建议，可不可以允许在后台修改重新连接次数或者每次尝试时间，我的电影文件比较大，如果不能修改链接时间或许永远看不了
![Screenshot_2024-04-14-15-27-23-405_com yujian ResideMenuDemo](https://github.com/alist-org/alist/assets/140718382/1fbee28e-8bc3-4833-a8f5-507c6a451ceb)

**Comment by pongfcnkl** [comment]
浏览器不支持你这个格式 需要自己转mp3
